src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
```

Comment:
here you can remove spaces
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
![loom link](https://github.com/mate-academy/layout_calendar/assets/79792776/5188d115-3207-445d-8345-0c39c1db8b48)
can't open